consensus/consensus_v2.go

```diff
@@ -205,6 +211,102 @@ func (consensus *Consensus) onAnnounce(msg *msg_pb.Message) {
 				Msg("[OnAnnounce] Block content is not verified successfully")
 			return
 		}
+
+		if consensus.ShardID == 0 {
+			//TODO: HB fix this
+			if recvMsg.BlockNum%core.ShardingSchedule.BlocksPerEpoch() == 0 {
+				consensus.generateNewVrf = true
```

Comment:
The validators don't need to have these flags (generateNewVdf/Vrf) because they don't generate them, they only verify them.

Reply:
removed these flags.